Dockerfile.jetson

```
@@ -0,0 +1,136 @@
+ARG L4T_MAJOR=35
+ARG L4T_MINOR=3
+ARG CUDA_MAJOR=11
+ARG CUDA_MINOR=4
+ARG CUDA_PATCH=19
+ARG ZED_SDK_MAJOR=4
+ARG ZED_SDK_MINOR=0
```

```suggestion
ARG ZED_SDK_MINOR=0
ARG ROS_DISTRO=noetic
```
Lets make this similar between dockerfiles